hooks/pre_start.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
-
-OPEN_FILES_RECOMMENDED=24576
```

Comment:
I would expect that if there is not any longer a recommended limit then [our infrastructure shall align](https://github.com/aeternity/infrastructure/blob/a759f5b1a12f49458146fb144b831882eb8fb57e/ansible/setup.yml#L150-L155)? If so, we may consider:
* Amending infrastructure repo removing ulimit limit.
* Provisioning fresh env (or at least fresh boxes in env) with such scripts (so that ulimit is left to default of OS/image).
* Smoke test such env.
* Load test such env e.g.: max peer number reached, sync handshake/ping happens when it should, rotating peers (hence causing potential TCP connections in TIME_WAIT state or similar).

Also we shall smoke test on supported OS/archs + Docker.